atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -329,6 +329,10 @@ def __init__(self, data, pulp_data, odcs_config, koji_tag=None, arches=None):
                 self.flags = [UNPUBLISHED_REPOS]
         self.koji_tag = koji_tag
         self.odcs_config = odcs_config
+        self.arches = arches or []
+        if data.get('with_multilib', False):
```

Comment:
Please add this key to the schema.

Reply:
from atomic_reactor/schemas/container.json
```
    "compose": {
      "type": "object"
    },
```
There's not much there to add to.

Reply:
It needs the properties definitions, like in https://github.com/projectatomic/atomic-reactor/pull/1056.

Needs definitions for packages (list of str), modules (list of str), signing_intent (str), and with_multilib (bool).

Reply:
Fixed.